include/swift/IDE/CodeCompletionCache.h

```diff
@@ -123,6 +124,7 @@ struct DenseMapInfo<swift::ide::CodeCompletionCache::Key> {
     H ^= std::hash<bool>()(Val.ResultsHaveLeadingDot);
     H ^= std::hash<bool>()(Val.ForTestableLookup);
     H ^= std::hash<bool>()(Val.ForPrivateImportLookup);
+    H ^= std::hash<bool>()(Val.AddInitsInToplevel);
```

Comment:
`CodeCompleteInitsInPostfixExpr` wasn't hashed 🤷

Reply:
Nice catches. Amusing that they were reversed (one in == and one in hash).